libs/dh/auth/feature-msal/src/lib/b2c-config.ts

```diff
@@ -55,6 +55,15 @@ export function MSALInstanceFactory(
   });
 }
 
+function reloadOnLoginFailed(error: string) {
+  const loginFailed = error.includes(
+    'Error - Guard - error while logging in, unable to activate'
```

Comment:
question: is this error message always in English? Also for browsers that are set to a different locale?

Reply:
@dzhavat yes the translation is an internal error message from the MSAL guard, so it's not a message for the users